.gitignore

```diff
@@ -7,4 +7,5 @@
 /server/server_win*
 /server/server_net*
 /server/server_ope*
+/server/server_lin*
```

Comment:
What is this for?

Reply:
This is because I use `gox` for building xplat binaries and it tags them with `{{name}}_{{os}}_{{arch}}`, so this just prevents them from getting accidentally added to the index.